### PR TITLE
Updates workloads map variable to flatten by type of workload (i.e. c…

### PR DESCRIPTION
**Pull request by jmarhee**
…ni) for networks, updates project creation flow to match other projects

Signed-off-by: Joseph D. Marhee <jmarhee@equinix.com>

### CODE REVIEW

**Comment by jmarhee** [comment]
I'm going to try to do the same for storage as I did networks in this PR as well. WIP

File: README.md [hunk 1]
```diff
@@ -30,7 +29,7 @@ Create a file called `main.tf` with the following contents:
 ```hcl
```
Comment (displague) [inline README.md:29]: An approach that seems appealing is to put these examples into an examples/ subfolder (allowing for different examples):

https://github.com/dsieczko/terraform-metal-windows-starter/tree/master/examples/simple

In the root README.md we can then refer folks to those example directories or the README.md in each example directory.

Comment (displague) [reply README.md:29]: fwiw, I don't think the Terraform registry does anything special with the examples.

Comment (displague) [reply README.md:29]: But this is the Hashicorp recommendation:
https://www.terraform.io/docs/language/modules/develop/structure.html
>  If you want to include an example for how this module can be used in combination with other resources, put it in an [examples directory like this](https://github.com/hashicorp/terraform-aws-consul/tree/master/examples). Consider including a visual diagram depicting the infrastructure resources the module may create and their relationship.

**Comment by displague** [comment]
Changes look good. In the future we may want to namespace the examples: examples/foo/main.tf

Cluster came up.

**Pull request merged**